e2e/playwright/pages/legal_hold_plugin.ts

```diff
@@ -21,6 +21,11 @@ export class LegalHoldPluginPage {
     readonly verifyStartDate: Locator;
     readonly verifyEndDate: Locator;
 
+    readonly verifyHoldOnPage: Locator;
+    readonly releasebutton: Locator;
```

Comment:
nit: can we follow the `camelCase` for all variables and attributes?

`releasebutton` -> `releaseButton`
`modalreleaseButton` -> `modalReleaseButton`

Reply:
got it